### DISCUSSION TITLE
GitHub Accelerator program Graduation Day

### DISCUSSION THREAD

**Discussion by thunder-coding** [termux/termux-packages#17110] [Announcements]
The GitHub Accelerator (https://accelerator.github.com/) program of 10 weeks for which Termux was selected for will have its Demo Day/Graduation on June 28, 2023. @agnostic-apollo will be presenting a 5 min demo on behalf of Termux at 9.40PDT/16.40GMT.

Grab your free (beer) tickets at https://www.eventbrite.com/e/github-accelerator-open-source-demo-day-tickets-662569482157 if you want to attend the live stream. The event should be recorded as well if you can't make it then.

You can also view the event live without having to get a free ticket on YouTube: https://www.youtube.com/watch?v=Gj6Bez2182k&t=7227s

A clipped version of Termux's presentation will be made available on [Termux's YouTube channel](https://youtube.com/@termuxdevs) as well in case you miss watching it live.

https://twitter.com/github/status/1673308399734251524
https://twitter.com/termuxdevs/status/1673501935725314048

The event will be starting in 2 hours as of creation of this discussion. Rumours also say that @agnostic-apollo will also do a face reveal on stream, so make sure to tune in!

**Comment by agnostic-apollo** [comment]
Just damn you!

**Reply by thunder-coding** [reply]
May I know why?

**Reply by agnostic-apollo** [reply]
Lolz, you know exactly why.

**Reply by thunder-coding** [reply]
*face reveal* right? 😁

**Reply by agnostic-apollo** [reply]
Duh!

**Reply by thunder-coding** [reply]
Maybe if you could bring that topic to GitHub staff. In case you get to keep your anonymity. :)

**Reply by agnostic-apollo** [reply]
It's not that much an issue at this point, just messing with you.

**Reply by thunder-coding** [reply]
At this point it's difficult to know if you're messing with me or being serious. Anyways maybe take an hour of rest, so you're fresh for your presentation. Make sure to keep an alarm, so you wake up on time.

**Reply by agnostic-apollo** [reply]
That's part of the whole messing with you! Rest was really not an option at the time ;)

**Comment by Dawimpy** [comment]
Congrats 🎉

**Comment by asumbek** [comment]
Stream is now live! Go watch it, go go go!

**Comment by X7md** [comment]
Oh yeah!
